collector/collectd/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM debian:stable-slim
+
+ENV DEBIAN_FRONTEND noninteractive
+COPY 50docker-apt-conf /etc/apt/apt.conf.d/
+
+COPY rootfs_prefix/ /usr/src/rootfs_prefix/
+
+RUN apt-get update \
+ && apt-get upgrade \
+ && apt-get install \
+    collectd-core \
+    collectd-utils \
+    build-essential \
+    zsh \
+ && make -C /usr/src/rootfs_prefix/ \
```

Comment:
is the rootfs stuff necessary? i'm not familiar enough w/ collectd to understand why it needs this